packages/@react-spectrum/table/docs/TableView.mdx

```diff
@@ -465,6 +465,28 @@ You can disable specific rows by providing an array of keys to TableView via the
 <PokemonTable selectionMode="multiple" disabledKeys={[3]} />
 ```
 
+### Highlight selection
+
+By default, TableView uses the checkbox selection style, which includes a column of checkboxes in each row for selection. When the `selectionStyle` prop is set to `"highlight"`, the checkboxes are hidden, and the selected rows are displayed with a highlighted background instead.
+
+In addition to changing the appearance, the selection behavior also changes depending on the `selectionStyle` prop. In the default checkbox selection style, clicking, tapping, or pressing the <kbd>Space</kbd> or <kbd>Enter</kbd> keys toggles selection for the focused row. Using the arrow keys moves focus but does not change selection.
```

Comment:
The following does seem accurate `Using the arrow keys moves focus but does not change selection.` Selection moves with focus and the up/down arrow keys. To move focus without changing selection you have to use a modifier key as explained in the next paragraph. Unless we are talking about left and right arrow keys because they are moving to cells of the row.

Reply:
You're thinking of highlight selection. It is describing checkbox selection: "In the default selection style, ...".